src/pages/Swap/AdvancedSwapMode/AdvancedSwapMode.component.tsx

```diff
@@ -0,0 +1,151 @@
+import { FC, PropsWithChildren, useRef } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Flex } from 'rebass'
+
+import { ButtonDark } from '../../../components/Button'
+import { useAdvancedTradingView } from '../../../services/AdvancedTradingView/useAdvancedTradingView.hook'
+import { useAllTrades } from '../../../services/AdvancedTradingView/useAllTrades.hook'
+import { TransactionRows } from '../../Account/TransactionRows'
+import {
+  AdvancedModeHeader,
+  AdvancedModeTitle,
+  AdvancedSwapModeContainer,
+  BaseWrapper,
+  ChartWrapper,
+  LiquidityWrapper,
+  OrderButton,
+  OrdersWrapper,
+  PairDetailsWrapper,
+  SwapBox,
+  TransactionsWrapper,
+} from './AdvancedSwapMode.styles'
+import { Chart } from './Chart'
+import { ColumnHeader } from './ColumnHeader'
+import { InfiniteScroll } from './InfiniteScroll'
+import { PairDetails } from './PairDetails'
+
+export const AdvancedSwapMode: FC<PropsWithChildren> = ({ children }) => {
+  const { t } = useTranslation('swap')
+
+  const tradesWrapper = useRef<HTMLDivElement>(null)
+
+  const {
+    transactions,
+    tradeHistory,
+    liquidityHistory,
+    hasMore: { hasMoreTrades, hasMoreActivity },
+  } = useAllTrades()
+
+  const {
+    chainId,
+    inputTokenSymbol,
+    outputTokenSymbol,
+    symbol,
+    showTrades,
+    isLoadingTrades,
+    isLoadingActivity,
+    fetchTrades,
+    fetchActivity,
+    pairTokens,
+    activeCurrencyOption,
+    handleAddLiquidity,
+    handleSwitchCurrency,
+    isFetched,
+  } = useAdvancedTradingView()
+
+  const [token0, token1] = pairTokens
+
+  return (
+    <AdvancedSwapModeContainer>
```

Comment:
We are starting to use the name container as a component with logic, might be better to not use it on describing styles.

Suggestion: 
```suggestion
    <AdvancedSwapModeGrid>
```
or even
```suggestion
    <AdvancedSwapModeWrapper>
```